app_link.module

```diff
@@ -7,6 +7,62 @@
  */
 
 /**
+ * Implements hook_help().
+ */
+function app_link_help($path, $arg) {
+  switch ($path) {
+    case 'admin/help#app_link':
+      $help_html = '';
+      $help_html .= '<p>' . t('This module will allow you to create empty placeholder pages in your Drupal site, that redirect to the appropriate App depending on the client that placeholder page is opened in.') . '</p>';
```

Comment:
Run on sentence.

This module will allow you to create empty placeholder pages in your Drupal site. These placeholder pages will redirect users to the appropriate App or App Store according to the user's device (iPhone, Android, …).